x-pack/test/security_solution_endpoint_api_int/apis/endpoint_artifacts/event_filter.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
```

Comment:
I've renamed this file to `event_filters.ts` and also moved host isolation tests to `endpoint_artifacts/host_isolations_exceptions.ts`